Packs/GoogleChronicleBackstory/Playbooks/playbook-ChronicleAsset_Hostname_And_IP_Address_Investigation_And_Remediation_-_Chronicle_README.md

```diff
@@ -0,0 +1,46 @@
+This playbook receives indicators from its parent playbook, performs enrichment and investigation for each one of them, provides an opportunity to isolate and block the Hostname or IP Address associated with the current indicator, and gives out a list of isolated and blocked entities.
```

Comment:
Playbook name should be "Hostname and IP Address Investigation and Remediation - ChronicleAsset" or just "Chronicle" in the end. the same for the other playbook.
Also in the sub-playbook description we should add "This playbook receives indicators from its parent playbook "<the name of the parent playbook"......"

Reply:
@altmannyarden We have applied these changes in [this](https://github.com/demisto/content/pull/10022/commits/6ed8556fa886b498aaeec84580c751fbc759eec9) commit.